acpi_tables/src/aml.rs

```diff
@@ -238,10 +238,131 @@ impl Aml for EISAName {
     }
 }
 
+fn create_integer(v: usize) -> Vec<u8> {
+    if v <= u8::max_value().into() {
+        (v as u8).to_bytes()
+    } else if v <= u16::max_value().into() {
+        (v as u16).to_bytes()
```

Comment:
Is that calling into the `to_bytes` implementation of `Word`? I assume it is.
There's a nightly `to_bytes` implementation for u16, u32 and u64. I wonder how this is going to work once it lands into a toolchain that we use.

Reply:
I think it should be fine as I think I read trait definitions take precedence. However to reduce the cognitive burden I added a commit that renames it to `to_am_bytes()`